src/misc/setup.cpp

```diff
@@ -1151,6 +1167,16 @@ bool CommandLine::FindInt(char const * const name,int & value,bool remove) {
 	return true;
 }
 
+bool CommandLine::FindIntOption(const std::string &name, int &value, bool remove) {
+	cmd_it it,it_next;
+	if (!IsOption(name, it))
+		return false;
+	it_next=it;++it_next;
+	value=atoi((*it_next).c_str());
+	if (remove) cmds.erase(it,++it_next);
+	return true;
```

Comment:
(lots of statements put in the same line, making it hard to easily read - `./scripts/format-commit.sh` can fix them up :+1: )

Reply:
Much of the code was copied from another function without change in my previous commits. But I have now run the said script to format such code too.